admin_app/src/utils/api.ts

```diff
@@ -60,7 +60,7 @@ const deleteContent = async (content_id: number, token: string) => {
 };
 
 const createContent = async (content: number, token: string) => {
-  return fetch(`${BACKEND_ROOT_PATH}/content/create`, {
+  return fetch(`${BACKEND_ROOT_PATH}/content`, {
```

Comment:
`createContent` is actually never used in the app and is a duplicate of `addContent`... We could remove either one, but I would actual vote for keeping `createContent` and removing `addContent` since "create" is what the endpoints are described as, not "add".

Reply:
Hmmm yeah. Create makes sense but we should probably also call `edit`, `update` then.
In the app we call the page `AddEditContentPage` 🤷🏽‍♂️ 

Anyway, went with your suggestion.